spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/repository/InMemoryStreamDefinitionRepository.java

```diff
@@ -122,4 +123,9 @@ public void deleteAll() {
 		definitions.clear();
 	}
 
+	@Override
+	public Page<StreamDefinition> search(SearchPageable searchPageable) {
```

Comment:
Have a feeling someone will ask for it.   If it is not handled here we will want a story for it.